src/http.rs

```diff
@@ -797,10 +797,10 @@ mod tests {
         template_context: TemplateContext,
     ) {
         let recipe = Recipe {
-            query: indexmap! {
-                "mode".into() => "sudo".into(),
-                "fast".into() => "true".into(),
-            },
+            query: QueryParameters::from(vec![
+                ("mode".into(), "{{mode}}".into()),
+                ("fast".into(), "true".into()),
+            ]),
```

Comment:
I think it'd be helpful to include a duplicate of `fast` in here, to make sure they both get filtered out.